### PR TITLE
databases-week1

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**

### CODE REVIEW

File: databases/week1/dbweek1.sql [hunk 1]
```diff
@@ -0,0 +1,111 @@
+-- Find out how many tasks are in the task table
+SELECT
+    COUNT(id)
```
Comment (lawansubba) [inline databases/week1/dbweek1.sql]: COUNT(id) as task_amount
_Pointer:_
Provide  a column name for columns you have created

File: databases/week1/dbweek1.sql [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+FROM 
+    task;
+
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT 
+    COUNT(id)
```
Comment (lawansubba) [inline databases/week1/dbweek1.sql]: COUNT(id) as task_amount
Pointer:
Provide a column name for columns you have created

File: databases/week1/dbweek1.sql [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+FROM 
+    task
+JOIN 
+    status
+ON 
+    task.status_id = status.id;
+
+-- Get the name of each status, along with a count of how many tasks have that status
+SELECT
+    status.name, COUNT(task.id)
```
Comment (lawansubba) [inline databases/week1/dbweek1.sql]: status.name, COUNT(task.id) as task_amount
_Pointer_
Provide Column names for columns you have created

File: databases/week1/dbweek1.sql [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    status
+JOIN
+    task
+ON
+    status.id = task.status_id
+GROUP BY status.name;
+
+-- Get the names of all statuses, sorted by the status with most tasks first
+SELECT
+    status.name, COUNT(task.id)
```
Comment (lawansubba) [inline databases/week1/dbweek1.sql]: status.name, COUNT(task.id) as task_amount

**Review by lawansubba — COMMENTED** [review]
Well done!! Only minor comments

File: databases/week1/dbweek1.sql (hunk 4 above)
Comment (SvitlanaBurlaka) [reply databases/week1/dbweek1.sql]: Thank you for review !

**Pull request merged**